packages/adapters/gupshup-whatsapp/src/gupshupWhatsappAdapter.spec.ts

```
@@ -195,15 +196,19 @@
+        metaData: {}  // We'll ignore sessionId in comparison
+      }
     };
     const xmsg = await adapter.convertMessageToXMsg(mockReport);
-    // Timestamp will be different every time, hence only
-    // check for existence of field.
     expect("timestamp" in xmsg).toBeTruthy();
     expect(xmsg.messageId.Id).toBeDefined();
     xmsg.timestamp = 0;
     xmsg.messageId.Id = 'testId';
+    if (xmsg.transformer?.metaData) {
+      delete xmsg.transformer.metaData.sessionId;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)